ChargedUp/src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -60,30 +61,33 @@ public class DriveSubsystem extends SubsystemBase {
   public double error;
 
 
-
+  private final SlewRateLimiter slewRateLimiter = new SlewRateLimiter(1);
   //solenoid to control gear shifting
 private DoubleSolenoid shiftSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, 2, 6              );
   public Boolean inHighGear = false;
   
   public Boolean BrakeMode = true;
   public WPI_Pigeon2 pigeon = new WPI_Pigeon2(DriveConstants.PIGEON_ID);
+  
   //odometry object 
   private final DifferentialDriveOdometry m_odometry;
   /** Creates a new DriveSubsystem. */
   public DriveSubsystem() {
     
     //back motors follow front motors
-    motorFR.setOpenLoopRampRate(.2); // 0.5 seconds from neutral to full output (during open-loop control)
+    motorFR.setOpenLoopRampRate(0); // 0.5 seconds from neutral to full output (during open-loop control)
```

Comment:
(I'm still learning how to use this tool. I should have added my other comment here) Line 78-87 are the lines that I believe the comments need to be updated. The value passed into the function changed but the comments didn't. I'm assuming the value passed into the method controls the time from neutral to full out.

Reply:
yes – that comment exists to explain how ramp rates work. In the actual code, the next line says  "0 disables ramping"
It has not been changed because ramps have a habit of being ctrl+c ctrl+v'd in from year to year – without understanding what they do. Last year that comment was added to the drivebase reusable master code to help explain it

Reply:
Understood, but that means the comment should not state "0.5 seconds from neutral to full output". It should instead state something like: the ramp timing is set in seconds with 0 being disabled and .5 being 1/2 second (or something). Copy/Paste of comments that aren't accurate to what the code actually does isn't a great idea. Providing guidance about the unit the value is (mSec, sec, min, hour, etc) in the comment is helpful. Also, this comment wouldn't need to be repeated 3 times on the subsequent 3 lines. (and yes, I'm being picky)